File: Java/Ejercicio/src/Ejercicio8.java
package Ejercicio.src;

import java.util.Scanner;

public class Ejercicio8 {

    public static void resultOperation(int number) {

        Double square = Math.pow(number, 2);
        Double squareRoot = Math.sqrt(number);
        Double cubeRoot = Math.pow(number, 1.0 / 3.0);

        System.out.println("\n tha oprationes of " + number + " are  \n");
        System.out.println("\n SQUARE IS = " + square + "\n");
        System.out.println("\n SQUARE ROOT IS = " + squareRoot + " \n ");
        System.out.println("\n CUBE ROOT IS = " + cubeRoot + "\n");
    }

    public static void main(String[] args) throws Exception {

        Scanner keyboard = new Scanner(System.in);
        System.out.println("please enter the number = ");
        int number = keyboard.nextInt();

        resultOperation(number);
        keyboard.close();
    }
}

File: Javascript/Ejercicio/Ejercicio8.js

    const resultOperation=(number)=>{

        const square = Math.pow(number,2);
        const squareRoot = Math.sqrt(number);
        const cubeRoot = Math.pow(number,1/3);

        process.stdout.write(`\n tha oprationes of ${number} are  \n`);
        process.stdout.write(`\n SQUARE IS = ${square}\n`);
        process.stdout.write(`\n SQUARE ROOT IS = ${squareRoot}\n`);
        process.stdout.write(`\n CUBE ROOT IS = ${cubeRoot}\n`);
    }


    process.stdout.write("please enter the number = ");
    process.stdin.on('data',async function(data1) {
      const number = await parseFloat(data1.toString());
      resultOperation(number)

  });
  
  
  